content/en/docs/tasks/debug-application-cluster/resource-metrics-pipeline.md

```diff
@@ -1,76 +1,201 @@
+
+To learn more about the Metrics API, see [resource metrics API design](https://github.com/kubernetes/design-proposals-archive/blob/main/instrumentation/resource-metrics-api.md), the [metrics-server repository](https://github.com/kubernetes-sigs/metrics-server) and the [resource metrics API](https://github.com/kubernetes/metrics#resource-metrics-api).
+
+
+{{< note >}} You must deploy the metrics-server or alternative adapter that serves the Metrics API to be able to access it. {{< /note >}}
+
+## Measuring resource usage
 
-{{< note >}}
-The API requires the metrics server to be deployed in the cluster. Otherwise it will be not available.
-{{< /note >}}
+### CPU
 
-## Measuring Resource Usage
+CPU is reported as the average core usage measured in cpu units. One cpu, in Kubernetes, is equivalent to 1 vCPU/Core for cloud providers, and 1 hyper-thread on bare-metal Intel processors.
 
-### CPU
+This value is derived by taking a rate over a cumulative CPU counter provided by the kernel (in both Linux and Windows kernels). The time window used to calculate CPU is shown under window field in Metrics API.
 
-CPU is reported as the average usage, in
-[CPU cores](/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu),
-over a period of time. This value is derived by taking a rate over a cumulative CPU counter
-provided by the kernel (in both Linux and Windows kernels).
-The kubelet chooses the window for the rate calculation.
+To learn more about the CPU, see [Meaning of CPU](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-cpu).
 
 ### Memory
 
-Memory is reported as the working set, in bytes, at the instant the metric was collected.
-In an ideal world, the "working set" is the amount of memory in-use that cannot be freed under memory pressure.
-However, calculation of the working set varies by host OS, and generally makes heavy use of heuristics to produce an estimate.
-It includes all anonymous (non-file-backed) memory since Kubernetes does not support swap.
-The metric typically also includes some cached (file-backed) memory, because the host OS cannot always reclaim such pages.
+Memory is reported as the working set, measured in bytes, at the instant the metric was collected.
+
+In an ideal world, the "working set" is the amount of memory in-use that cannot be freed under memory pressure. However, calculation of the working set varies by host OS, and generally makes heavy use of heuristics to produce an estimate. 
+
+It includes all anonymous (non-file-backed) memory since Kubernetes does not support memory swap. The metric also includes some cached (file-backed) memory, because the host OS cannot always reclaim pages.
+
+To learn more about memory, see the [Meaning of memory](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-memory).
```

Comment:
```suggestion
To learn more about how Kubernetes allocates and measures memory resources, see [meaning of memory](/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-memory).
```
(although, since the definition we're linking to unsurprising in terms of metrics, it might be OK to omit this paragraph).